lib/Core/Search/Common/SiteAccessConfigResolver.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Netgen\IbexaSearchExtra\Core\Search\Common;
+
+use Ibexa\Contracts\Core\Persistence\Content\Handler as ContentHandler;
+use Ibexa\Contracts\Core\Persistence\Content\Location\Handler as LocationHandler;
+use Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException;
+use RuntimeException;
+
+class SiteAccessConfigResolver
```

Comment:
Rename to `SiteConfigResolver` (each site config can map multiple siteaccesses).